src/components/layout/PanelChange/index.tsx

```diff
@@ -115,7 +115,7 @@ export const PanelChange: FC<IProps> = ({
   }, [web3]);
 
   function getFormattedNumber(num: string) {
-    return parseFloat(num).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+    return parseFloat(num).toString().replace(/\B(?=(\d{3})+(?!\d))/g, '');
```

Comment:
don't change here bro, it will break for other values. just remove the `getFormattedNumber` method where we don't want.

Reply:
I'll try to get that fixed today